src/components/taskDetails/Details.tsx

```diff
@@ -2,12 +2,12 @@ import React, { FC } from 'react';
 import setColor from './taskPriorityColors';
 import classNames from './task-details.module.scss';
 import { TaskDetailsProps } from '@/interfaces/taskDetails.type';
-import extractRepoName from '@/utils/extractRepoName';
+import { transformGitHubApiUrl, extractRepoName } from '@/utils/urlTransform';
 
 const Details: FC<TaskDetailsProps> = ({ detailType, value }) => {
     const color = value ? setColor?.[value] : undefined;
     const isGitHubLink = detailType === 'Link';
```

Comment:
If there are multiple types of detail type, we should create a structure that could hold potential values and then use that to equate rather than hard coding.